src/main/fc/fc_tasks.c

```diff
@@ -502,16 +475,7 @@ cfTask_t cfTasks[TASK_COUNT] = {
     },
 #endif
 
-#ifdef USE_RANGEFINDER
```

Comment:
Why this Task got removed?

Reply:
Range finder didn't really work, and neither did altitude hold on betaflight in its current state, so the use of the rangefinder in the calculateEstimatedAltitude function was removed along with the ifdefs.